guide/src/advanced/features.md

```diff
@@ -124,7 +124,8 @@ __Feature comparison between Hermes and the Go relayer__
 |                        |       |        |
 | Modular Architecture   | ✅    | ✅     | defined interface can be implemented for different chain types
 | Cl_Non_Tendermint      | ❌    | ❌     | supports non tendermint IBC light clients
-| Chain_Non_Cosmos       | ❌    | ✅     | supports non cosmos-SDK chains
+| Chain_Non_Cosmos       | ✅    | ✅     | supports non cosmos-SDK chains
```

Comment:
```suggestion
```

Reply:
Done in https://github.com/informalsystems/hermes/pull/4297/commits/74a1e7dfc7395a31ea2e66069e56ea02f27e3171